modules/tensor_mechanics/tests/linear_elastic/tests

```diff
@@ -1,10 +1,4 @@
 [Tests]
-  [./tween]
-    type = 'Exodiff'
-    input = 'tween.i'
```

Comment:
OK - but eventually you will remove it. I can live with that. I'm just concerned that it'll get out of date quickly.